docs/ADRs/015-use-cypress-audit-for-pa11y.md

```diff
@@ -0,0 +1,36 @@
+# 015 — Use Cypress Audit to run pa11y tests
+
+This is an update to [ADR 005 — Define Frontend Toolset for Accessibility Standards](./005-frontend-a11y-toolset.md). When that ADR was written we were still using Nightwatch and not Cypress. Now that we are using Cypress, it makes sense to use [Cypress Audit](https://github.com/mfrachet/cypress-audit) as the tool for running pa11y tests against our app.
```

Comment:
awesome! Can you remove the dependencies for pa11y-ci, the `test:a11y` command, and pa11y-ci config file and replace the readme documentation for `Technologies.md` around pa11y-ci